Tests/FuzzilliTests/LifterTest.swift

```diff
@@ -736,6 +736,30 @@ class LifterTests: XCTestCase {
 
         XCTAssertEqual(lifted_program,expected_program)
     }
+
+    func testRegExpInline(){
+        let fuzzer = makeMockFuzzer()
+        let b = fuzzer.makeBuilder()
+
+        let v0 = b.loadRegExp("a", RegExpFlags(rawValue:0))
+        b.compare(v0,v0, with: Comparator.equal);
```

Comment:
Nit: I think you can drop `Comparator` here and below: `b.compare(v0,v0, with: .equal);`